src/Modules/Frontend/Domain/Block/AbstractBlockController.php

```diff
@@ -90,7 +90,7 @@ public function __invoke(Request $request, Response $response): string|array
         return $this->twig->render($this->templatePath, $this->assignedContent);
     }
 
-    final protected function assign(string $key, mixed $value): void
+    final public function assign(string $key, mixed $value): void
```

Comment:
shouldn't be public

Reply:
https://github.com/forkcms/forkcms/pull/3555/commits/c24fac573aad3d9e429b9a352c24087b2dd1f862